.github/workflows/build.yml

```diff
@@ -145,11 +145,19 @@ jobs:
   build-msvc-arm:
     strategy:
       matrix:
-        toolset: ['14','14.2'] # VS 2022, 2019
+        os: [windows-2022, windows-latest]
+        toolset: ['14', '14.29'] # VS 2022 (ongoing), 2019 (last)
         arch: [arm, arm64]
         config: [Debug, Release]
+        exclude:
+          # arm64: windows-latest is enough/fine.
+          - os: windows-2022
+            arch: arm64
+          # arm (sdk): only available on windows-2022.
```

Comment:
These comments seem to be backwards.

Reply:
The comment for the arm64 block says “windows-latest is enough/fine”, yet the block below the comment says the OS is `windows-2022`. Vice versa for the arm block right below it.

Reply:
You know what, you’re right. For some reason I never saw the `exclude:` part. Sorry to bother you!